### DISCUSSION TITLE
Release 5.2.0 Soon?

### DISCUSSION THREAD

**Discussion by jcburley** [golang-jwt/jwt#364] [General]
Given the recent security update to `x/net`, we're looking to release some of our (Internet-facing) Golang services with all the pending upgrades ASAP.

As we need the recent change to export `Validator`, is a Release contemplated anytime soon (like, real soon), so we can finally get our pending JWT v4->v5 upgrade off our backs and catch two birds with one trap, so to speak?

Just wondering...and hopeful that the answer is (say) "yes, by Monday Dec 4th".

**Comment by mfridman** [comment]
Ye sure, we can cut a release. I'll do it today or tomorrow, cc @oxisto fyi

**Comment by mfridman** [comment]
Done, hopefully this unblocks you. https://github.com/golang-jwt/jwt/releases/tag/v5.2.0

**Reply by jcburley** [reply]
Looking good so far — just merged and released our core Golang package with this upgrade, so there'll likely be a bunch of Renovate PRs generated for our various Golang services to upgrade that package.

Thanks!!
